parser_tests/update_snapshots.py

```diff
@@ -0,0 +1,68 @@
+import json
```

Comment:
Will add the shebang. I would like to keep the script close to the tests, though. It should only rarely and very conciously be used, since an updated snapshot is tautological in the sense that the tests will always pass. We should probably write some documentation on it to inform users of how to use it.

Reply:
You are right, this shouldn't be called too often, but I still think we should have some top-level script to run and manage the snapshots. We can add this later.

However, one or two sentences in the readme about this feature would be nice! :wink:

Reply:
I don't know much about best practices with python scripts. It's just for the argument of discoverability that I would rather mention it in documentation instead of putting it at the root.

But if there are other reasons to put it in the root, e. g. that being best or common practice, then please tell me so. :)

I will also update the README.md to educate about parser development.